test/specs/validate-spec/validate-spec.spec.js

```diff
@@ -2,186 +2,350 @@ const { expect } = require('chai');
 const OpenAPIParser = require('../../..');
 const path = require('../../utils/path');
 
-describe('Invalid APIs (Swagger 2.0 specification validation)', () => {
```

Comment:
There's a lot of changeset noise in this file but the gist of what I did:

* Change the `tests.forEach()` testing flow over to using `describe` for each test case.
* Extended each test to also test OpenAPI 3.0, OpenAPI 3.1, or OpenAPI 3.x when appropriate.